src/components/views/rooms/MessageComposer.tsx

```diff
@@ -436,6 +438,17 @@ export default class MessageComposer extends React.Component<IProps, IState> {
             );
         }
 
+        const msgComposerClassNames = classNames(
+            "mx_MessageComposer",
+            {
+                // IRC layout has nothing for message composer so use group layout stuff
+                // "mx_IRCLayout": this.props.layout == Layout.IRC,
+                // "mx_GroupLayout": this.props.layout == Layout.Group,
```

Comment:
I don't think I understand the purpose of these comments

Reply:
The IRC layout uses the group layout for the message composer because it has no own rules for that, the bubble layout has. I left them in sperated to easily be able to adapt this if that changes.

Reply:
Ah, sorry. I more of meant why is there the commented code? 

```
// "mx_IRCLayout": this.props.layout == Layout.IRC,
// "mx_GroupLayout": this.props.layout == Layout.Group,
```

Reply:
If the irc layout gets its own rules for the message composer those lines can be commented in and the line after those two removed. It was just for remeberance once that might happen.

Reply:
Ah, ok. I think that if this changes at any time, it probably won't be a big problem to add these lines, so it's not necessary to keep them there. If you're keen on keeping it, something like this might be better:
```suggestion
				// When IRC layout gets something for the message composer we can use the following
                // "mx_IRCLayout": this.props.layout == Layout.IRC,
                // "mx_GroupLayout": this.props.layout == Layout.Group,

                // IRC layout has nothing for message composer so use group layout stuff
```

Reply:
Whatever, I don't really care. 🙃